references/recognition/README.md

```diff
@@ -64,3 +64,15 @@ Feel free to inspect the multiple script option to customize your training to yo
 ```python
 python references/recognition/train_pytorch.py --help
 ```
+## Using custom fonts
+If you want to use your own custom fonts for training, make sure the font is installed on your OS. 
+You can do so in linux by either:
+* clicking on the .ttf file and pressing the install button.  
+* If the above doesn't work, copy the .ttf file to the desired directory with: ```sudo cp BORING.ttf /usr/local/share/fonts/``` and then run ```fc-cache -f -v``` to build the font cache.
+
```

Comment:
Mh .. Keep in mind that passing fonts to the training script will only work with the wordgenerator, so if you pass paths to your custom dataset will not change your Images (anything like this 😅 )

Reply:
Yep sounds better 👍 If no path to a dataset is passed like `python3 doctr/references/recognition/train_pytorch.py crnn_mobilenet_v3_small --vocab french`  it "triggers" the WordGenerator to create random images from the given vocab

Reply:
Great, will replace the second line with what you mentioned after the thumbs up!.

Reply:
Mmmh, on Linux (Ubuntu at least), you can install fonts by clicking on them similarly to Windows :)

Reply:
this does not work or better PIL does search only in usr/local/.. and not in ~./fonts so the described way is the one who will work in each case :)

Reply:
oh, good to know :pray: